language.json

```diff
@@ -694,11 +694,11 @@
         "scanner": {
           "title": "Søk etter node eller objekt",
           "search": "Søk",
-          "noMatchingNodeOrObject": "Fant ingen node eller objekt for %{barcode}",
-          "noMatchingNode": "Fant ingen node for %{uuid}",
-          "cannotActOnObject": "Kan bare reagere på object via magasin eller objekt plukkliste",
-          "cannotActOnNode": "Kan bare reagere på node fra magasin, node plukkliste eller flyttedialog",
-          "noCurrentLocation": "Objektet er ikke plassert"
+          "noMatchingNodeOrObject": "Fant ingen node eller objekt med strekkode %{barcode}.",
+          "noMatchingNode": "Fant ingen node med UUID %{uuid}.",
```

Comment:
Good points. But we dont have any other identifier in this case. We dont know anything other than the provided UUID or barcode.